### ISSUE TITLE
Event and job can easily be mixed up when generating a report

### ISSUE THREAD

**Issue by maydhak** [systers/vms#196]
When a report is generated for a volunteer, it is easy to get an event and a job mixed up since they are not selectable options. So if a field is typed in incorrectly, it could mess the whole report up.

**Comment by smarshy** [comment]
@tapasweni-pathak Dropdowns for volunteer report have been implemented. For admin, jobs are missing - This is relating to issue  #276. Once that is done, there wouldn't be any mix up, I guess and then this can be closed.

**Comment by tapasweni-pathak** [comment]
As we have another issue #276 for a specific case of this issue, it is better if I close this. Great work. :+1:
